submissions/vovan-zt/TiniWorld2/index.js

```diff
@@ -0,0 +1,42 @@
+class WorldInhabitant {
+   constructor(species, name, gender, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+      this.prop = [species, name, gender, saying];
+   }
+
+   getMeaning() {
+      return this.prop.map(meaning => meaning);
```

Comment:
What is the purpose of `.map` here?

Reply:
Last time you asked me to optimize the code

getMeaning() {
      return super.getMeaning() + `${this.legs}; ${this.hands}; ${this.friends}`;

As a peer developer on the project I am tasked with adding 10 more properties to humans. Please optimize this fragment so that I'd need as small amount of keystrokes to introduce these new properties as only possible. Ideally, I'd like to just list new properties in this method.

I did it like this. If it's bad I can change it

Reply:
.map didn't make any sense there